modules/admin_manual/nav.adoc

```diff
@@ -97,16 +91,22 @@
 **** xref:configuration/server/harden_server.adoc[Harden Server]
 **** xref:configuration/server/import_ssl_cert.adoc[Import SSL Cert]
 **** xref:configuration/server/index_php_less_urls.adoc[Index PHP Less URLs]
+**** xref:configuration/server/occ_command.adoc[OCC Command]
```

Comment:
why did you put occ_command and reverse_proxy out of the alphabetical sort order of the `****` list level?

Reply:
occ_command is correct. Yes,  reverse_proxy_configuration is wrong indeed. Sorry I missed it.